### DISCUSSION TITLE
Exporting plot to matplotlib

### DISCUSSION THREAD

**Discussion by dupsys** [JasonKessler/scattertext#124] [General]
Hi all,
Try to explore content of a scattertext to to matplotlib after installing  textalloc==0.0.3 and matplotlib>=3.6.0  but got this error:AttributeError: 'Figure' object has no property 'width_ratios'

Here is my code:

scattertext_structure = st.produce_scattertext_explorer(corpus1,
                    category='positive',
                    category_name='positive',
                    not_category_name='negative',
                    minimum_term_frequency=5,
                    pmi_threshold_coefficient=8,
                    width_in_pixels=1000,
                    return_scatterplot_structure=True,
           )
fig = st.produce_scattertext_pyplot(scattertext_structure)
fig.savefig(path +'tweetpyplot_export.png', format='png', dpi=500)

looking forward to your help. Thanks

**Comment by JasonKessler** [comment]
Could you retry this in a fresh virtual environment with the newest version of Scattetext installed? That could rule out a dependency issue.

If you're still having problems, could you include a code sample that I can paste in and run to reproduce the issue? I can't manipulate `corpus1`, and it's possible there's something special about that object which is causing issues.
